src/routes/blogRoute.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
```

Comment:
Remove this exception. We don't want our users to see data logged in the console all the time.